tools/db_bench_tool.cc

```diff
@@ -2349,6 +2392,16 @@ class CombinedStats {
     return avg;
   }
 
+  double CalcConfidence95(std::vector<double> data) {
```

Comment:
This parameter (and others including prior to your change) should be `const std::vector<double>& data`